app/models/ability.rb

```diff
@@ -244,6 +247,7 @@ def signed_in_with_volunteers_coordinator_role(user)
     # ids of all the conferences for which the user has the 'volunteers_coordinator' role
     conf_ids_for_volunteers_coordinator = Conference.with_role(:volunteers_coordinator, user).pluck(:id)
 
+    can [:index, :show, :update], Resource, conference_id: conf_ids_for_coordinator
```

Comment:
And that's why you should first write some tests for the new abilities you are introducing :)